CONTRIBUTING.md

```diff
@@ -115,28 +144,231 @@ our project's repositories. The [Developer Certificate of Origin
+ If your PR fixes any issues, the description should refer to it (e.g., "fixes #NNN") or
+ you can associate it using "linked issues". Either of these links your PR to the issue
+ and automatically closes the issue when the PR is accepted.
+ Do **not** use the PR comments to document the code as they will most likely never be
+ seen after the PR is merged. Please **do** make your code self-explanatory. If there is
+ information that is needed by a programmer reading the source code, you should put it in
+ a code comment. This also applies to answering questions from reviewers: it is best done
+ by clarifying the code or add documentation, rather than in the PR comment thread.
+
+1. A reviewer will assign themselves to the pull request. If you don't see anyone
+ assigned after 3 business days, you can leave a comment asking for a review, or ping in
+ slack. Sometimes we have busy days, sick days, weekends and vacations, so a little
+ patience is appreciated! 🙇‍♀️
+
+1. The reviewer will leave feedback.
+
+* `nits`: These are suggestions that you may decide to incorporate into your pull
+ request or not without further comment.
+* Requests for change in the PR contents. These require resolution before the PR is
+ merged.
+* It is okay to clarify if you are being told to make a change or if it is a suggestion.
+
+If you agree with a code review comment and do what it suggests, don't respond in the
+ GitHub code review system. Simply resolve the conversation. Respond if more discussion
+ is needed, such as asking a follow-up question or explaining why you disagree with the
+ suggestion. If the reviewer is asking a question, then usually the best way to answer
+ it is by improving the code or documentation. Answering it only in the code review
+ will not help future programmers after the PR is merged.
+
+Respond to the feedback by making changes in your working copy, committing them, and
+ periodically pushing them to GitHub when the tests pass locally. As soon as you
+ receive feedback, you can start working on it. The reviewer should assign the code
+ review back to you, but they might forget, so don't wait for that.
+
+After you have made the changes (in new commits please!), leave a comment asking the
+ the reviewer to take another look. If 3 business days go by with no review, it is okay
+ to bump. With the exception of rebasing discussed below, please do **not** force a push
+ with `git push -f` as it might cause loss of code review comments and context associated
+ with previous commits.
+
+When the PR is ready for merging, you may need to rebase your PR on top of the current codebase.
+This can be done via the PR's GitHub Web page or with the CLI as either `git pull --rebase`
+or `git fetch` and `git rebase` from `upstream/main`.
+If there are any conflicts please fix them locally before committing and pushing to the PR branch.
+If you're unsure on how to resolve a conflict, please ask the reviewer for guidance or help.
```

Comment:
I would remove this.

Reply:
clarification: remove just the last sentence, regarding resolving the conflict?

Reply:
The last sentence I would remove.
As for the block, I would remove it as I think it is trivial.